code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -178,32 +164,31 @@ This saves us from having to call add_fingerprint() any time something is put in
 			return BP_R_ARM
 
 
-/mob/living/carbon/human/equip_to_slot(obj/item/W, slot, redraw_mob = 1)
+/mob/living/carbon/human/equip_to_slot(obj/item/Item, slot, redraw_mob = 1)
```

Comment:
why use Item here?

It's commonplace everywhere to use an uppercase I for a local variable like this. It prevents potential namespace pollution of more common words that are often used for object variables